js/lib/nfc/Arab.js

```diff
@@ -1,7 +1,7 @@
 /*
  * Arab.js - include file for normalization data for a particular script
  * 
- * Copyright © 2013 - 2018, JEDLSoft
+ * Copyright © 2013 - 2018, 2020 JEDLSoft
```

Comment:
2020?

Reply:
updated the tool to generate the right copyright notice